Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxMask/TextBoxMask.Properties.cs

```diff
@@ -97,7 +98,7 @@ public static string GetCustomMask(DependencyObject obj)
         /// </summary>
         /// <param name="obj">TextBox Control</param>
         /// <param name="value">CustomMask Value</param>
-        public static void SetCustomMask(DependencyObject obj, string value)
+        public static void SetCustomMask(TextBox obj, string value)
```

Comment:
Can you change the sample to ensure that we have use both x:Bind and Binding.. I have a feeling TextBox is going to bust x:Bind